src/Modules/RotatableModule.cpp

```diff
@@ -0,0 +1,17 @@
+/*
+** EPITECH PROJECT, 2024
+** OpenThisFuckingDoor
+** File description:
+** Rotatable
+*/
+
+#include <RotatableModule.hpp>
+#include <MovementModule.hpp>
+
+RotatableModule::RotatableModule(GameObject *gameObject)
+{
+    if (gameObject->hasModule<RotatableModule>())
+        throw std::runtime_error("GameObject already has a RotatableModule!");
```

Comment:
Define an error class in the Module class an then throw this type of error like this :"throw Error("msg..")"